tools/policy_template_release_generation/generate_policy_release_notification_contents.rb

```diff
@@ -88,5 +88,5 @@ def initialize(name, path)
 end
 
 # Output Notification Content as a JSON string to be used directly in YAML workflow file
-all_notification_content = JSON.generate(all_notification_content_array).gsub('"', '\\"')
+all_notification_content = JSON.generate(all_notification_content_array).gsub('"', '\\"').gsub('\\\"', '\\\\\\\\\\"')
```

Comment:
Took a bit of trial and error, but this produces the expected results i.e., three backslashes preceding the double quote. For example:
`[{\"activityTitle\":\"<h2 style='font-size: 18px;'>Azure Unused Volumes</h2>\",\"facts\":[{\"name\":\"Template Version\",\"value\":\"6.0\"},{\"name\":\"Policy Updates\",\"value\":\"<ul><li>Several parameters altered to be more descriptive and human-readable</li><li>Removed deprecated \\\"Log to CM Audit Entries\\\" parameter</li><li>Removed ability to filter by how long a policy was detached; see README for more details</li><li>Added ability to filter by volume age</li><li>Added ability to only report recommendations that meet a minimum savings threshold</li><li>Added ability to filter resources by multiple tag key:value pairs</li><li>Added additional context to incident description</li><li>Normalized incident export to be consistent with other policies</li><li>Added human-readable recommendation to incident export</li><li>Policy no longer raises new escalations if ages or savings data changed but nothing else has</li><li>Streamlined code for better readability and faster execution</li></ul>\"}]}]`

Open to changing this to something more elegant, but this works (strangely)